tests/Composer/Test/Command/SelfUpdateCommandTest.php

```diff
@@ -0,0 +1,60 @@
+<?php declare(strict_types=1);
+
```

Comment:
```suggestion
/*
 * This file is part of Composer.
 *
 * (c) Nils Adermann <naderman@naderman.de>
 *     Jordi Boggiano <j.boggiano@seld.be>
 *
 * For the full copyright and license information, please view the LICENSE
 * file that was distributed with this source code.
 */
```

Maybe we should add the above suggestion for consistency with the other tests?

Reply:
@theoboldalex 
Thanks for let me know this !
I forgot this discription😓